Makefile

```diff
@@ -123,13 +130,13 @@ test: manifests generate diskmaker-rbac fmt ## Run tests.
 
 ##@ Deployment
 
-install: manifests diskmaker-rbac kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
+install: manifests rbac kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/crd | kubectl apply -f -
 
-uninstall: manifests diskmaker-rbac kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
+uninstall: manifests rbac kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
```

Comment:
Useless to fix what you remove altogether in the next commit.

Reply:
True, but I would still like to keep this change in this commit, so at least it refers to a valid Makefile target (even though it's removed by the next commit).